### ISSUE TITLE
Problems executing shell scripts on a windows based nodejs installation

### ISSUE THREAD

**Issue by LordOfTheDings** [kevinfaveri/solana-candy-factory#23]
Hi
I've been having some issues while trying to set up a NFT playground using the candy factory. 
After registering my nft on the strapi server and generating the nft sources, I can't execute the "yarn setup-dev" command.
Always getting this error message:

**The command 
 "sh" is either misspelled or
could not be found.**

The problem arises in the setup-metaplex-dev.js file when trying to execute the shell script:

shelljs.exec('sh .devtools/dev/setup-metaplex-dev.sh')

I've already tried fixing this by adding the git/bin directory to my enironvment variables in order to make "sh" known to Windows, without success though.

I'd be happy for some help on this one :)

**Comment by kevinfaveri** [comment]
I will leave this issue, but I must say that I do not have the faintest idea because this project was meant to be used with UNIX based environments.

**Comment by LordOfTheDings** [comment]
I've since managed to set up a Linux environment using WSL2 and an Ubuntu Installation (18.04). 
However, now VSCode is denied the execution of the commands within the shell scripts.

I tried to resolve this with the solutions most commonly suggested on the internet: 

`$ sudo chown -R <user-name> <directory-name>` and
`sudo chmod 755 <directory-name>`

VSCode is able to "execute" the files, but since the commands themselves are still blocked I end up in the same place as before :(

![image](https://user-images.githubusercontent.com/55738329/135722164-a2f584ad-2917-4039-bb76-83056e1cf335.png)

![image](https://user-images.githubusercontent.com/55738329/135722157-6bbfbdcc-e6cb-4731-ace9-7ee602d05c2f.png)

Any ideas on how to solve this?
